packages/core/src/YouVersionPlatformConfiguration.ts

```diff
@@ -1,12 +1,9 @@
-import type { YouVersionUserInfo } from './YouVersionUserInfo';
-
 export class YouVersionPlatformConfiguration {
```

Comment:
While `localStorage` is standard for SPAs, it's vulnerable to XSS attacks. We could document XSS prevention best practices:

```suggestion
/**
 * Security Note: Tokens are stored in localStorage for persistence.
 * Ensure your application follows XSS prevention best practices:
 * - Sanitize user input
 * - Use Content Security Policy headers
 * - Avoid innerHTML with untrusted content
 */
export class YouVersionPlatformConfiguration {
```